src/App.js

```diff
@@ -2,46 +2,89 @@ import React from "react";
 import "./App.css";
 import { makeShuffledDeck } from "./utils.js";
 
-class App extends React.Component {
-  constructor(props) {
-    // Always call super with props in constructor to initialise parent class
-    super(props);
-    this.state = {
-      // Set default value of card deck to new shuffled deck
-      cardDeck: makeShuffledDeck(),
-      // currCards holds the cards from the current round
-      currCards: [],
-    };
+export default function App(props) {
+  const [cards, setCards] = React.useState({
```

Comment:
I think you should define multiple different states, instead of having one state named "cards" and storing everything in it. The variable name is misleading then, as well as the state management gets very tricky!

Reply:
Example:

```suggestion
  const [cards, setCards] = React.useState(makeShuffledDeck())
  const [currentCards, setCurrentCards] = React.useState([])
  const [totalCards, setTotalCards] = React.useState(52)
```